convert2rhel/actions/pre_ponr_changes/backup_system.py

```diff
@@ -77,8 +80,57 @@ def run(self):
 
         super(BackupRepository, self).run()
 
-        repo.backup_yum_repos()
-        repo.backup_varsdir()
+        self.backup_yum_repos()
+
+    def backup_yum_repos(self):
+        """Backup .repo files in /etc/yum.repos.d/ so the repositories can be restored on rollback."""
+        loggerinst.info("Backing up .repo files from %s." % DEFAULT_YUM_REPOFILE_DIR)
+
+        repo_files_backed_up = False
+
+        for repo in os.listdir(DEFAULT_YUM_REPOFILE_DIR):
+            if repo.endswith(".repo") and repo != "redhat.repo":
```

Comment:
As far as I can tell we don't test when there's a redhat.repo right?

Reply:
We are not, I'll add the unit test for this case